pegasus/sites.v3/code.org/public/css/hoc2022-landing-page.scss

```diff
@@ -145,37 +145,6 @@ header {
 
 main {
 
-  // NASA skinny banner
```

Comment:
I moved all of the sass code to a separate vanilla css file ([/css/hoc2022-nasa-skinny-banner.css](https://github.com/code-dot-org/code-dot-org/blob/52b92807c28d327b376e7cda68a0029d3a57d0c1/pegasus/sites.v3/code.org/public/css/hoc2022-nasa-skinny-banner.css) or see below) so we can more easily update this in the coming weeks through Dropbox.

Reply:
Good call!